keyboards/chlx/str_merro60/str_merro60.c

```diff
@@ -0,0 +1,28 @@
+/* 
+Copyright 2021 Alexander Lee <chlx.bsmt@gmail.com> 
+
+This program is free software: you can redistribute it and/or modify 
+it under the terms of the GNU General Public License as published by 
+the Free Software Foundation, either version 2 of the License, or 
+(at your option) any later version. 
+
+This program is distributed in the hope that it will be useful, 
+but WITHOUT ANY WARRANTY; without even the implied warranty of 
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+GNU General Public License for more details. 
+
+You should have received a copy of the GNU General Public License 
+along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+*/ 
+
+#include "str_merro60.h"
+
+// Borrowed from hineybush h88
+void eeconfig_init_kb(void) {  // EEPROM is getting reset!
+  rgblight_enable(); // Enable RGB by default
+  rgblight_sethsv(0, 255, 128);  // Set default HSV - red hue, full saturation, medium brightness
+  rgblight_mode(RGBLIGHT_MODE_RAINBOW_SWIRL + 2); // set to RGB_RAINBOW_SWIRL by default
```

Comment:
These are not needed, as there are defines to handle this. 

https://docs.qmk.fm/#/feature_rgblight?id=configuration

Reply:
removed eeprom reset code in *.cadded RGBLIGHT_DEFAULT_MODE and specified RGBLIGHT_EFFECT_* in place of RGBLIGHT_ANIMATIONS in